Doc/library/socket.rst

```diff
@@ -674,6 +674,14 @@ Constants
 
   .. availability:: Linux >= 3.9
 
+.. data:: SO_REUSEPORT_LB
+
+   Constant to enable duplicate address and port bindings with load balancing.
```

Comment:
Should `SO_REUSEPORT_LB` be preferred on FreeBSD instead of `SO_REUSEPORT`?

Reply:
Yeah, the original ticket is to add the flag then use it in socket.create_connection, I thought I'd split it into two PRs for easier review

Reply:
I'm not sure if it could he helpful but we could mention that it should be preferrable to use it instead of `SO_REUSEPORT` then (I'm not sure whether we should recommend what to use in the docs).